hibike/hibike_message.py

```diff
@@ -432,6 +434,43 @@ def read(serial_conn):
 
   return HibikeMessage(messageID, payload)
 
+# Sends this message
+# Computes the checksum
+# Then sends each byte of the message, and finally sends the checksum byte
+def hid_send(hid, message):
```

Comment:
See my above comment.  Untill it's tested, I'm not convinced that there should be a packet limit size anymore.